fetch/corb/script-html-via-cross-origin-blob-url.sub.html

```diff
@@ -19,7 +19,7 @@
   }
 
   function step2_processSubframeMsg(msg) {
-    assert_not_exists(msg, 'error');
+    assert_false(msg.haOwnProperty('error'), 'unexpected property found: "error"');
```

Comment:
`haOwnProperty`

Reply:
Oof, yes. Also, I introduced a syntax error in testharness.js. Both are fixed now